.github/workflows/check-pr-scalars.yml

```diff
@@ -0,0 +1,77 @@
+name: Check PR scalars
+
+on:
+    workflow_dispatch:
+    push:
+        branches: ['main']
+        paths:
+            - 'packages/scalars/**'
+    pull_request:
+        branches: ['main']
+        paths:
+            - 'packages/scalars/**'
+        types: [opened, synchronize]
+
+env:
+  NX_ISOLATE_PLUGINS: false
+
+jobs:
+    lint:
+        name: Lint
+        runs-on: ubuntu-latest
+        container: node:22
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install PNPM
+              uses: pnpm/action-setup@v3
+              with:
+                  version: latest-8
+                  run_install: false
+
+            - name: Install Dependencies
+              run: pnpm install --no-frozen-lockfile
```

Comment:
I think it makes sense to use `--frozen-lockfile` as it's enabled by default in CI. Any reason not to?
It's important to enforce the lockfiles to be updated.